app/src/main/java/swen_anigans/mathematicfanatic/QuestionContent.java

```diff
@@ -0,0 +1,73 @@
+package swen_anigans.mathematicfanatic;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Created by Sean on 11/15/2016.
+ */
+
+public class QuestionContent
+{
+    protected ArrayList<Question> questions;
+
+    //region constructors
+
+    public QuestionContent(int totalPages)
+    {
+        // set to temporarily hold our questions
+        Set<Question> uniqueQuestions = new HashSet<>();
+
+        // get all the possible numbers we can use to make a question.
+        ArrayList<Integer> numbers = new ArrayList<Integer>();
+        int min = DataManager.getInstance().curStudent.rangeMin;
+        int max = DataManager.getInstance().curStudent.rangeMax;
+        for(int i = min; i <= max; i++)
+        {
+            numbers.add(i);
+        }
+
+        // using a set to create our questions makes sure we don't have duplicates
+        while(uniqueQuestions.size() < totalPages)
+        {
```

Comment:
It seems to still work. It just uses the "secondNumber" (B) in the help supplier. Which is what I think help_activity was doing before.